public/res_secondaryLanguage.json

```
@@ -1,80 +1,52 @@
       "section_name": {
-        "about": "O mnie",
-        "projects": "Projekty",
+        "about": "Über mich",
+        "projects": "Projekte",
         "skills": "Umiejętności",
-        "experience": "Doświadczenie"
+        "experience": "Doświadczenie",
+        "recommendations": "Empfehlungen"
       }
     },
     "projects": [
       {
-        "title": "Animal Shelter",
-        "startDate": "2020",
-        "description": "Najbardziej rozbudowana aplikacja nad którą miałam okazję pracować. Poznałam dużo technologii a mój kod by sprawdzany przez super opiekuna. Aplikacja obsługuje wszystkie procesy adopcji i pozwala na przechowywanie wszelkiej dokumentacji związanej z adopcją zwierząt ze schroniska.",
+        "title": "Todo App",
+        "startDate": "2021",
+        "description": "Ich lernte eine Menge über SOLID Prinzipien während dieses Odin bootcamp Projektes. Ich liebe es organisiert zu sein (Ein Hoch auf GTD!). Deshalb baue ich auch gerade gemeinsam mit einem befreundetem Entwickler eine erweiterte Todo App - mit Firebase backend, und zusätzlicher Funktionalität. Um die Architektur zu optimieren, erstellten wir ein UML Diagramm.",
         "images": [
-          "images/portfolio/animal-shelter/p1.jpg",
-          "images/portfolio/animal-shelter/p2.jpg"
+          "images/portfolio/todo-app/todo-896-504.png",
+          "images/portfolio/todo-app/todo2-896-504.png"
         ],
-        "url": "",
+        "url": "https://davidmatalik.github.io/todo/",
         "technologies": [
           {
-            "class": "devicon-angularjs-plain",
-            "name": "Angular"
-          },
-          {
-            "class": "devicon-typescript-plain",
-            "name": "TypeScript"
+            "class": "devicon-javascript-plain",
+            "name": "JavaScript"
           },
           {
-            "class": "devicon-csharp-plain",
-            "name": "C#"
+            "class": "devicon-webpack-plain",
+            "name": "Webpack"
           }
         ]
       },
       {
-        "title": "Photography",
-        "startDate": "2018",
-        "description": "Osobisty projekt na przedmiot ze studiów. Byłam odpowiedzilna za testowanie aplikacji, która optymalizuje zdjęcia z pomocą popularnych algorytmów używanych przed edytory grafiki takie jak Pixlr czy Adobe Photoshop. Otrzymałam 5-tkę za projekt :)",
+        "title": "Wetter App",
+        "startDate": "2021",
+        "description": "Dieses Projekt half mir mein Verständnis von APIs und asynchronen Funktionen zu vertiefen. Mit dieser Wetter App kann man nach jeder Stadt auf der Welt suchen und bekommt die aktuellen Wetterdaten mit passenden Wetter icons geliefert.",
```

```suggestion
        "description": "Dieses Projekt half mir, mein Verständnis von APIs und asynchronen Funktionen zu vertiefen. Mit dieser Wetter App kann man nach jeder Stadt auf der Welt suchen und bekommt die aktuellen Wetterdaten mit passenden Wetter icons geliefert.",
```

glaube ich :grimacing:

---

Glaube auch ;)